client/public/index.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
+
+use Symfony\Component\Debug\Debug;
+use Symfony\Component\Dotenv\Dotenv;
+use Symfony\Component\HttpFoundation\Request;
+
+require dirname(__DIR__).'/vendor/autoload.php';
+require dirname(__DIR__) . '/app/AppKernel.php';
+
+// We don't want to load the .env files if we are running on a server - ENVIRONMENT is only injected via terraform
+if (empty($_SERVER['ENVIRONMENT'])) {
+    (new Dotenv())->load(dirname(__DIR__).'/frontend.env', dirname(__DIR__).'/admin.env');
+}
+
+if ($_SERVER['APP_DEBUG']) {
+    umask(0000);
+
+    /**
+     * Excluding notices as they include array undefined index errors which we rely on, e.g. :
+     *
+     * $ary['test'] = null;
+     *
+     * Trying to access $ary['test'] would throw a notice error and app error
+     */
+    Debug::enable(E_ALL & ~E_NOTICE);
+}
+
+if ($trustedProxies = $_SERVER['TRUSTED_PROXIES'] ?? $_ENV['TRUSTED_PROXIES'] ?? false) {
+    Request::setTrustedProxies(explode(',', $trustedProxies), Request::HEADER_X_FORWARDED_ALL ^ Request::HEADER_X_FORWARDED_HOST);
+}
+
+if ($trustedHosts = $_SERVER['TRUSTED_HOSTS'] ?? $_ENV['TRUSTED_HOSTS'] ?? false) {
+    Request::setTrustedHosts([$trustedHosts]);
+}
+
+$_SERVER += $_ENV;
+$_SERVER['APP_ENV'] = $_ENV['APP_ENV'] = ($_SERVER['APP_ENV'] ?? $_ENV['APP_ENV'] ?? null) ?: 'dev';
```

Comment:
Future you will probably hate having to parse what this chain of ternary operators is doing. Might be worth breaking it out to be more readable.

Reply:
Agree - I'd taken this from some official Symfony files but even as I was adding it I had to manually parse through in a REPL to make sure it was doing what I thought it was.